test/units/TEST-74-AUX-UTILS.busctl.sh

```diff
@@ -53,6 +53,12 @@ busctl emit --auto-start=no --destination=systemd-logind.service \
             /org/freedesktop/login1 org.freedesktop.login1.Manager \
             PrepareForShutdown b false
 
+systemd-run --service-type=notify --unit=test-busctl-wait busctl --timeout=3 wait /test org.freedesktop.fake1 TestSignal
+busctl emit /test org.freedesktop.fake1 TestSignal s success
+
+(! systemctl is-failed test-busctl-wait.service )
```

Comment:
The service may not be stopped yet. For safety, better to use wait for termination with `timeout` command:
```
timeout 10 bash -c 'while systemctl is-active test-busctl-wait.service; do sleep .1; done'
(! systemctl is-failed test-busctl-wait.service )
```